src/components/Button/index.tsx

```diff
@@ -15,12 +15,14 @@ interface IButtonProps {
 }
 
 export const Button: React.FC<IButtonProps> = (props) => {
-    const { options } = props;
-    const { value, handlerClick, iconName } = options;
+    const {
+        options: { value, handleClick, iconName },
+    } = props;
+
     const classnames = classNames(iconName, 'game-buttons__icon');
```

Comment:
isn't it better to make it as props property by making default method in used place?

Reply:
What do you mean?

Reply:
i mean this 
classnames should be renamed to classNames - consistency
and should be moved to the component with buttons, not in button, as button component shouldn't have context

Reply:
Thanks, makes sense!